go.mod

```diff
@@ -138,3 +140,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/openshift/api => github.com/openshift/api v0.0.0-20230327113148-36ce464529eb
```

Comment:
Why replacing? I guess it would be better to update the version.

Reply:
accidentally committed this, updated the version as requested

Reply:
Thanks